rider/migrations/0001_initial.py

```diff
@@ -9,23 +9,23 @@ class Migration(SchemaMigration):
 
     def forwards(self, orm):
         # Adding model 'Rider'
-        db.create_table('rider_rider', (
+        db.create_table(u'rider_rider', (
```

Comment:
In Python, it makes it a [Unicode string](https://docs.python.org/2/tutorial/introduction.html#unicode-strings).  Not sure why it's required for the migration.